pkg/Microsoft.ML/Microsoft.ML.nupkgproj

```diff
@@ -20,5 +20,7 @@
   <ItemGroup>
     <Content Include="..\common\CommonPackage.props" Pack="true" PackagePath="build\netstandard2.0\$(MSBuildProjectName).props" />
     <Content Include="build\**\*" Pack="true" PackagePath="build" />
+    <Content Include="..\..\LICENSE" Pack="true" PackagePath="\" />
```

Comment:
We shouldn't need this file in the .nupkg itself because we are already linking to this license file in the .nuspec metadata file.

https://github.com/dotnet/machinelearning/blob/7fb76b026d0035d6da4d0b46bd3f2a6e3c0ce3f1/pkg/Directory.Build.props#L24

Reply:
The metadata has this link but if I download the package and unzip it, finding the license becomes difficult. Do you think this is too much to include?

Reply:
We can include it if you want. I won't hurt anything.

But similar to the below thread, I think it should be included in *all* nuget packages in our repo, not in just one.

This should be added in https://github.com/dotnet/machinelearning/blob/d68388a1c9994a5b429b194b64b2b0782834cb78/pkg/Directory.Build.props#L32-L36 instead.